code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -8,11 +8,11 @@
 	icon_state = "red"
 	item_state = "toolbox_red"
 	obj_flags = OBJ_FLAG_CONDUCTIBLE
-	force = 20
-	attack_cooldown = 21
+	force = 12
+	attack_cooldown = 23
 	melee_accuracy_bonus = -20
-	throwforce = 10
-	
+	throwforce = 14
```

Comment:
Why is this needed? People are just going to pick baseball bats instead to do their B&Es then, and metal can commonly be found in lockers. Why shouldn't an iron toolbox do rather well as a reserve weapon? As it isn't sharp, it poses little-to-no threat to necromorphs.

Reply:
20 was a pretty crazy amount of force, most of the good weapons in our code have about 15 damage, higher values are bay holdovers we haven't gotten around to rebalancing.

A toolbox is a heavy object but not really intended as a weapon, its shape isn't amenable for efficient swinging. A baseball bat would make more sense